### PR TITLE
State and function updates for speed and sub pagination.

### PR DESCRIPTION
**Pull request by KellyInnovation**

### CODE REVIEW

**Comment by JMStewart00** [comment]
Doing work!

File: lfucg-exactions/client/js/components/SubdivisionSummary.js [hunk 1]
```diff
@@ -40,98 +40,103 @@ class SubdivisionSummary extends React.Component {
         
         return (
             <div className="subdivision-summary">
-                <Navbar />
+                {!!subdivisions && !!subdivisions.currentSubdivision && <div>
```
Comment (JMStewart00) [inline lfucg-exactions/client/js/components/SubdivisionSummary.js]: @KellyInnovation Just thought I would point out something that Matt showed me with stuff like this... Destructuring. Basically giving react something as initial state to go off of to keep it from breaking
In the this.props declaration I believe you can give subdivisions an initial value like `subdivisions = []` and then below that you can do something like `cont { currentSubdivision = {} } = subdivision` and just call currentSubdivision down in the body as needed with no dot syntax. Thought you'd find it interesting. Keeps you from having to do all the && logic and what not. Maybe you knew that already though.

File: lfucg-exactions/client/js/components/AccountExisting.js [hunk 2]
```diff
@@ -55,7 +55,7 @@ class AccountExisting extends React.Component {
                 };
             })(lots));
 
-        const accounts_list = accounts && accounts.length > 0 ? (
+        const accounts_list = !!accounts && !!accounts.accounts && accounts.accounts.length > 0 ? (
```
Comment (JMStewart00) [inline lfucg-exactions/client/js/components/AccountExisting.js]: @KellyInnovation Whats the significance behind the `!!`?

Comment (KellyInnovation) [reply lfucg-exactions/client/js/components/AccountExisting.js]: It is a way to make sure it isn't false (null, undefined, empty in some way)

**Pull request merged**